extensions/hints/init.lua

```diff
@@ -114,7 +149,14 @@ function hints.displayHintsForDict(dict, prefixstring, showTitles)
         else
           local suffixString = ""
           if showTitles then
-            suffixString = ": "..win:title()
+            win_title = win:title()
+            if hints.titleMaxSize > 1 then
+                end_idx = hints.titleMaxSize - 6
+                if end_idx < 6 then end_idx = 6 end
+                win_title = string.sub(win_title, 1, end_idx)
+                if end_idx < #win_title then win_title = win_title .. "..." end
```

Comment:
You already modified `win_title` in the previous line so won't this condition never activate?